src/AbstractDriver.php

```diff
@@ -117,13 +117,13 @@ public function webservice($name, WebserviceInterface $webservice = null)
      *
      * @param string $name The registry alias for the webservice instance
      * @param \Muffin\Webservice\Webservice\WebserviceInterface $webservice Instance of the webservice
-     * @return $this
+     * @return \Muffin\Webservice\Webservice\WebserviceInterface
      */
     public function setWebservice($name, WebserviceInterface $webservice)
     {
         $this->_webservices[$name] = $webservice;
 
-        return $this;
+        return $this->_webservices[$name];
```

Comment:
Why did you change this? Returning the webservice instance which was passed as argument is not very useful. All setter methods should return `$this` to maintain consistency and intuitiveness

Reply:
I got confused by myself because I had made a mistake when refactoring.

[This line](https://github.com/davidyell/Webservice/blob/refactor/mixed-mode-methods/src/Model/Endpoint.php#L733) was using a call to `setWebservice()` via the Connection, and the Connection proxies to the Driver. So I'd updated it to preserve the return.

However, when I double checked my work, I realised that I should have used a getter, and obviously forgot to update this.